agent-svcat/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM quay.io/feedhenry/jenkins-agent-base-centos7:latest
+
+MAINTAINER Vitalii Chepeliuk <vchepeli@redhat.com>
+
+USER root
+
+ARG SVCAT_VER=0.1.21
+ARG SVCAT_BIN=svcat
+ARG SVCAT_URL=https://download.svcat.sh/cli/v${SVCAT_VER}/linux/amd64/${SVCAT_BIN}
+ARG MBCLI_VER=1.0.0
+ARG MBCLI_BIN=mobile
+ARG MBCLI_GZIP=mobile-cli_${MBCLI_VER}_linux_amd64.tar.gz
+ARG MBCLI_URL=https://github.com/aerogear/mobile-cli/releases/download/${MBCLI_VER}/${MBCLI_GZIP}
+ENV USER user
+ENV PASSWD passwd
+ENV URL url
+
+RUN yum -y update && \
+    yum -y install \
+        curl \
+        unzip \
+        wget \
+        zip && \
+    yum clean all
+
+RUN cd /tmp && curl -s -O -J -L ${SVCAT_URL} && chmod +x ${SVCAT_BIN} && mv ${SVCAT_BIN} /usr/local/bin/${SVCAT_BIN}
+RUN cd /tmp && curl -s -O -J -L ${MBCLI_URL} && tar -xf ${MBCLI_GZIP} && mv ${MBCLI_BIN} /usr/local/bin/${MBCLI_BIN}
+
+ENTRYPOINT oc login --insecure-skip-tls-verify=true -u ${USER} -p ${PASSWD} ${URL}
```

Comment:
@maleck13 @mikenairn the reason why we have the oc login as an entrypoint here is that mobile-cli requires ~/.kube/config file to exist on the slave.
When using the OpenShift pod as a Jenkins slave, serviceaccount jenkins is logged in via oc, but the config is not there, this is the way how we can create it.

Reply:
@psturc  `--insecure-skip-tls-verify=true` should be replaced with `--certificate-authority=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`

Reply:
@psturc Login should use the service account token for Jenkins SA at `/var/run/secrets/kubernetes.io/serviceaccount/token`